css/properties/mix-blend-mode.json

```diff
@@ -39,7 +39,7 @@
               "version_added": null
             },
             "webview_android": {
-              "version_added": "41"
+              "version_added": false
```

Comment:
Actually, chrome_android should have been corrected to 41.

Reply:
Separated as #4761, reverting this change!